README.md

```diff
@@ -25,6 +35,36 @@ We hypothesize that creating a new rules service could help shift the space towa
 - **[Standards](https://github.com/18F/eligibility-rules-service/issues/17)** - What are the most important areas to align around, standards to adhere to, and patterns to develop to make adoption the easy choice, without being too prescriptive?
 - **[Team](https://github.com/18F/eligibility-rules-service/issues/18)** - What team would we need for the following build/borrow phase?
 
+## To be validated
+
+#### :white_check_mark: _Validated:_ TTS has a role to play in the eligibility space
+_From Phase 1 Conclusions (longform):_ "TTS is in a unique position to be a leader, influencing the ecosystem and its actors from a variety of angles due to our ability to work with agencies across the federal government, our experience supporting states through procurement consulting, our ongoing engagement with the vendor community, and our ability to build products and platforms for use across government."
```

Comment:
If we mention these docs, I wonder if we want to include their contents in the wiki, and link to those wiki pages here? If not, it feels weird to pose these as quotes from another doc - we could just ditch the quotation marks in that case, since these are our own words (though I'm def in favor of adding those docs to the repo because #transparency 🙂)

Reply:
Yes, I would like to as well. I wanted to check in with you first since they we're out there yet. It's probably worth giving them a once-over to make sure it's all good to go.

Reply:
Ok yeah, good call. I can try to look back through them later this afternoon! If we wanted to get the other changes merged ASAP though, we could technically cut the reference/quotation marks now and merge without them, and then once we've looked at the other docs and have posted them to the wiki, we could add the references (and links) back in then? But also totally cool if you'd rather wait & do it all at once.

Reply:
I have time to do this now, no worries. I'll take a look and add them here before we merge.